code/modules/mob/living/brain/posibrain.dm

```diff
@@ -23,11 +23,6 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	var/fail_message = "<span class='notice'>The positronic brain buzzes quietly, and the golden lights fade away. Perhaps you could try again?</span>"
 	///Role assigned to the newly created mind
 	var/new_role = "Positronic Brain"
-	///Message sent to the player possessing the brain
-	var/welcome_message = "<span class='warning'>ALL PAST LIVES ARE FORGOTTEN.</span>\n\
```

Comment:
dont you need to keep a default message if no policy is set

Reply:
idk cobby told me to delete it

Reply:
i told headmins they need to update it